apps/member-profile/app/routes/_profile.companies_.$id.tsx

```diff
@@ -46,6 +46,7 @@ export async function loader({ params, request }: LoaderFunctionArgs) {
         'companies.imageUrl',
         'companies.name',
         'companies.levelsFyiSlug',
+        'companies.leetcodeTagged',
```

Comment:
Since we're changing the name to leetcodeTaggedSlug in the database, we need to have that changed reflected here

Reply:
Done